requirements.txt

```diff
@@ -6,6 +6,6 @@ matplotlib
 numba
 numpy
 numpy-quaternion
-pillow
+pillow<=8.3.1
```

Comment:
Forcing requirements because of wheel dependencies is a silly and if you HAVE to do it, then use the magic python version comments.

Reply:
Thanks for the pip magic.